index.html

```diff
@@ -1037,9 +1037,8 @@ <h3>Concrete Lifecycle Example</h3>
 Implementers that are interested in understanding more about the
 <code>proof</code> mechanism used above can learn more in
 Section <a href="#proofs-signatures"></a> and by reading the
-following specifications: Linked Data Proofs [[LD-PROOFS]], Linked Data
-Signatures [[LD-SIGNATURES]], 2018 RSA Signature Suite [[LDS-RSA2018]], and
-JSON Web Signature (JWS) Unencoded Payload Option [[RFC7797]]. A list
+following specifications: Linked Data Proofs [[?LD-PROOFS]], 
+Linked Data Cryptographic Suites Registry[[?LDP-REGISTRY]], and JSON Web Signature (JWS) Unencoded Payload Option [[RFC7797]]. A list
```

Comment:
This breaks the 75 character formatting fence, please fix. We need to be very deliberate with these changes since we're changing a global standard.